### ISSUE TITLE
PANTHER:PTN000075014

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#2200]
lid2 | Lid2 complex PHD finger and jmjC subunit Lid2 |   | DNA-binding transcription repressor activity, RNA polymerase II-specific |   | GO_Central | Schizosaccharomyces pombe | IBA | MGI:MGI:104813PANTHER:PTN000075014 | jumonji domain containing protein pthr10694 | protein |   | PMID:21873635 | 20180301

Lid2 is a histone demethylase
(gets catalytic activity, then 
DNA-binding transcription repressor activity, RNA polymerase II-specific from this)

**Comment by ValWood** [comment]
Also

jmj3 | Lid2 complex subunit, histone demethylase H3-K36 specific Jmj3 |   | DNA-binding transcription repressor activity, RNA polymerase II-specific |   | GO_Central | Schizosaccharomyces pombe | IBA | MGI:MGI:104813PANTHER:PTN000075014 | jumonji domain containing protein pthr10694 | protein |   | PMID:21873635 | 20180301

**Comment by ValWood** [comment]
jmj2	histone demethylase Jmj2		DNA-binding transcription repressor activity, RNA polymerase II-specific		GO_Central	Schizosaccharomyces pombe	IBA	MGI:MGI:104813
PANTHER:PTN000075014	jumonji domain containing protein pthr10694	protein		PMID:21873635	20180301


msc1 | Swr1 complex subunit Msc1 | Schizosaccharomyces pombe | jumonji domain containing protein pthr10694
-- | -- | -- | --

**Comment by pgaudet** [comment]
SPBP19A11.06 is annotated to 'negative regulation of transcription by RNA polymerase II' - you think that's unique to that one ?

**Comment by ValWood** [comment]
I don't think SPBP19A11.06 should have this annotation, but that depends how  heterochromatin  assembly and transcription are related. 

It is involved in chromatin silencing at the centromere and this is positioned under "negative regulation of transcription". I'm not sure we should have these 'silencing'  terms (they are probably a readout), they are legacy terms before we know what the mechanism of silencing really was.... but this needs wider discussion.

It will be difficult to make inferences across species for processes for these gene products.

Basically they are involved in heterochromatin spreading (at least this one is). I don't know if that means that it should be annotated to transcription though....

**Comment by pgaudet** [comment]
How about chromatin remodeling ?

**Comment by pgaudet** [comment]
@ValWood Should SPAC343.11c be annotated to BP histone acetylation ? It doesn't match the MF.

**Comment by ValWood** [comment]
I think that would be safest.

**Comment by ValWood** [comment]
> Should SPAC343.11c be annotated to BP histone acetylation ? It doesn't match the MF.

should probably be "regulation of histone acetylation" 
or  probably even
GO:0035034    histone acetyltransferase regulator activity
It binds to the acetylase, is a member of an acetylase complex and affects "histone acetylation"

However, I guess we aren't sure yet whether its activity in histone exchange is directly related to  the action of MSc1 on the histone acetyltransferase activity OR it has an independent function. Histone acetylase might only be affected indirectly because the complex does not form properly in the mutant.  

I'm never really sure how to annotate these...
I think there is enough information to say it is involved in Histone acetylation, but not enough to know if that role is direct, or as a regulator ....

Maybe "regulation of histone acetylation" would be better here?

**Comment by ValWood** [comment]
Actually, for these modification in process terms, it makes sense that if it has a role in the process, and it isn't the *actual* acetylase, then it *must* be regulating the process?

**Comment by ValWood** [comment]
Oh shoot, its wrong *Anyway* all of the above should be "DEacetylation" not "acetylation"!!!

**Comment by pgaudet** [comment]
yes sorry, we have deacetylase activity in PAINT.

**Comment by ValWood** [comment]
I deleted that annotation. I would not annotate to "deacetylase activity", that's a function of the complex this subunit doesn't have the activity.

Swr1 complex is a multimember, multifunctional complex, but it isn't the deacetylase.


It's possibly regulates the process. For now, I deleted the "histone acetylation" and I didn't replace it with "regulation of histone acetylation" becasue this is really old annotation, it hasn't been through Canto yet.
I suspect that we would probably do  "regulation of histone acetylation" for this.....
